libopenage/gui/guisys/private/gui_subtree_impl.cpp

```diff
@@ -1,8 +1,10 @@
 // Copyright 2015-2017 the openage authors. See copying.md for legal info.
 
 #include "gui_subtree_impl.h"
+#include "gui_renderer_impl.h"
 
 #include <cassert>
+#include <ciso646>
```

Comment:
http://en.cppreference.com/w/cpp/language/operator_alternative

> Because in C++ these are built into the language, the C++ version of <iso646.h>, as well as <ciso646>, does not define anything.

-> no need to replace them with symbol operators and no need to include the header